src/scenes/Partnerships/Create/CreatePartnership.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Except } from 'type-fest';
+import {
+  CreatePartnership as CreatePartnershipType,
+  GQLOperations,
+} from '../../../api';
+import { OrganizationLookupItem } from '../../../components/form/Lookup';
+import { PartnershipForm, PartnershipFormProps } from '../PartnershipForm';
+import { useCreatePartnershipMutation } from './CreatePartnership.generated';
+
+export interface CreatePartnershipFormInput {
+  partnership: Pick<
+    CreatePartnershipType,
+    'projectId' | 'types' | 'fundingType'
+  > & {
+    organizationId: OrganizationLookupItem;
+  };
```

Comment:
I'm assuming you are just trying to change the type of `organizationId`. If that's the case, `Merge` from `type-fest` would probably better indicate your intentions and doesn't require you to list the inverse set explicitly.
```tsx
Merge<CreatePartnershipType, { organizationId: OrganizationLookupItem }>
```

Reply:
I was also narrowing the field types to leave out several that we allow in `EditPartnership` but don't allow here

Reply:
Got it, no worries then.